index.bs

```diff
@@ -203,8 +230,27 @@ spec: mediaqueries-4; urlPrefix: https://drafts.csswg.org/mediaqueries-4/#
       </p>
 
       <p>
-        A <dfn>valid VideoConfiguration contentType</dfn> MUST include a media
-        type, subtype and full codecs description.
+        To check if a {{VideoConfiguration}} <var>configuration</var> is a
+        <dfn>valid video configuration</dfn>, the following steps MUST be run:
+        <ol>
+          <li>
+            If <var>configuration</var> is <code>null</code>, return
```

Comment:
I don't think the step is unreachable because the parameter is another dictionary. I guess the right term is "not present". Updated.